<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWgtTip</class>
 <widget class="QWidget" name="CWgtTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标签编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QComboBox" name="m_cmbVessel"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>病变信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>闭塞程度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbBSCD"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>病变范围</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbBBFW"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>病变类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbBBLX"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>病变形态</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbBBXT"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>钙化程度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbGHCD"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>TIMI血流</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_cmbTIMI"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
